### ISSUE TITLE
Light telemetry support 

### ISSUE THREAD

**Issue by marksev1** [raul-ortega/amv-open360tracker#1]
I would like for lighttelemetry to be supported. It is a low baudrate protocol at 1200 or 2400 baud it transmitts all the needed telemetry for an antenna tracker and more. It is already supported in Taulabs and has a PR ready for merge in cleanflight. Ezgui is also currently implementing it. There is also a gcs and osd supporting it available. I can send protocol specification ane links to CF source etc.

Samuel already said he is going to support it in the future, but if you are maybe faster you can open a PR at his repository.

Info about protocol: https://github.com/cleanflight/cleanflight/issues/36
PR Cleanflight: https://github.com/cleanflight/cleanflight/pull/1010

**Comment by raul-ortega** [comment]
It could be possible. Could you share some stream in hexadecimal format???

**Comment by marksev1** [comment]
I currently have my test setup in pieces since I'm upgrading something, but I will try to provide the raw stream to you from some other guy. 

This ltm is good since its so low baudrate that it can be used with openlrsng transparent serial telemetry and have less an effect on range vs higher baudrate protocols.

Also I've read on your forum Aeromodelismovirtual that some guys are using mavlink, for them "Ghettoproxy" on github would be useful since it converts mavlink to ltm (runs on a arduino promini, which you can have connected to pixhawk). There is also Ghettostation antenna tracker there which support ltm (and I have built one). But I want to try your antenna tracker maybe also, since it self-calibrates and stuff.

**Comment by raul-ortega** [comment]
Binary data is enough for me. Do you know which flight controller/OSD systems are using LTM?

**Comment by raul-ortega** [comment]
Some user use mavlink protocol transmitted over orangerx/openlrs without additional devices.

**Comment by marksev1** [comment]
Taulabs uses it, and cleanflight - is marked to be merged into next release (also cleanflight nav-rewrite branch).

**Comment by raul-ortega** [comment]
Ok, by now I have enough information to start, this afternoon I'm going to start implementing the parser, and if possible also the decoder part to produce valid frames.

**Comment by marksev1** [comment]
I was reading on aeromodelismo, Simba was asking if it need an aditional arduino on board. No need, for cleanflight (once PR is merged) and cleanflight inav nav-rewrite branch - this one has improved PH and RTH over regular cleanflight. And it is already working in Taulabs firmware also (on all of their boards). So for those you don't need an additional arduino on board.  Only for arducopter/arduplane, you need an aditional pro mini. But even that might change I will ask them if they are interested in inplementing ltm as a secondary telemetry protocol besides mavlink.

This video below (mwp_2015-11-08_145805) is done using ltm: http://www.rcgroups.com/forums/showpost.php?p=33143115&postcount=915

**Comment by raul-ortega** [comment]
Of course, I know that with cleanfligh/baseflight the additional arduino won't be necesary...they ask because the don't have NAZE32 based flight controller, they use other systems and are asking if the could get some benefit with LTM.

**Comment by marksev1** [comment]
Ltm decoded file (stronnag):
[ltm_decoded.txt](https://github.com/raul-ortega/amv-open360tracker/files/36959/ltm_decoded.txt)
Link to raw ltm log (from stronnag): https://drive.google.com/file/d/0B87FK33qoAT3UE9TcnpCcm55MTA/view?usp=sharing

**Comment by raul-ortega** [comment]
Ok, I've got both.

**Comment by marksev1** [comment]
Thanks Raul!

**Comment by raul-ortega** [comment]
The decoder is almost ready, but I need to know Lat, Lon, Alt for comparing with the results...

**Comment by marksev1** [comment]
Here is text from stronnag's mail: "The raw dump contains two flights, the first is the PH / RTH flight.
Please see
http://www.rcgroups.com/forums/showpost.php?p=33143115&postcount=915
for a _video_ replay of the flight. It includes GPS Hold and RTH
events, so a bit more interesting than yesterday."

So I think you could get your relevant coordinates from the video mwp_2015-11-08_145805 in this  post.

**Comment by raul-ortega** [comment]
This is what I get after decoding (lat and lon as int32 type): 

I only need to transform Lat, Lon to the correct format, but the video dosn't fit with this information I've got.

Lat: 1045975768, Lon: 521520172, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045975593, Lon: 521520387, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045982650, Lon: 521519999, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045982088, Lon: 521519556, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981701, Lon: 521520925, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981891, Lon: 521521284, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981846, Lon: 521521283, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981835, Lon: 521521330, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981782, Lon: 521521412, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981883, Lon: 521521180, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981708, Lon: 521521329, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981723, Lon: 521521317, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045981972, Lon: 521521315, Alt: 0, Sats: 17, SatsFix: 71
Lat: 1045982035, Lon: 521521381, Alt: 0, Sats: 17, SatsFix: 71
....

**Comment by marksev1** [comment]
Aha actually when I re-read I might have sent you the wrong ltm file. I think I sent you mwp raw log which is not the right file.

Attaching another one. This is probably the right ltm dump
[raw_dump.txt](https://github.com/raul-ortega/amv-open360tracker/files/37724/raw_dump.txt)

Sorry Raul.

**Comment by raul-ortega** [comment]
To convert the int32 lat/long to decimal Lat/Lon, I use the same method as in https://github.com/KipK/Ghettostation/blob/0530118e1f11556e226d6166544d8f51a423b295/GhettOSD/GhettOSD/Lighttelemetry.cpp

if (LTMcmd==LIGHTTELEMETRY_GFRAME)
  {

```
osd_lat = (int32_t)ltmread_u32() / 10000000.0;
osd_lon = (int32_t)ltmread_u32() / 10000000.0;
...
osd_alt = ((int32_t)ltmread_u32()) / 100.0f;      // altitude from cm to m.
```

   ...
  }

And I got the same results with the new raw file:

Lat/Lon: 104.597572, 52.152016, Alt: 2371747.840000, Sats: 17, SatsFix: 71
Lat/Lon: 104.597565, 52.152039, Alt: 2374369.280000, Sats: 17, SatsFix: 71
Lat/Lon: 104.598267, 52.152000, Alt: 2374369.280000, Sats: 17, SatsFix: 71
Lat/Lon: 104.598206, 52.151955, Alt: 2374369.280000, Sats: 17, SatsFix: 71
Lat/Lon: 104.598167, 52.152092, Alt: 2371747.840000, Sats: 17, SatsFix: 71
Lat/Lon: 104.598190, 52.152126, Alt: 2374369.280000, Sats: 17, SatsFix: 71
Lat/Lon: 104.598183, 52.152126, Alt: 2374369.280000, Sats: 17, SatsFix: 71
Lat/Lon: 104.598183, 52.15...
....

**Comment by marksev1** [comment]
Does it matter that this was done with 2400 baudrate? Probably I'm going to need to call stronnag to the discussion :) to figure this out.

**Comment by stronnag** [comment]
Hi,
May I suggest you use the original tar file I sent marksev1, 
http://www.zen35309.zen.co.uk/mwp-ltm/ltm_2015-11-08.tar.gz

This contains, inter alia, the raw byte stream and a hex listing, with a README that explains what the data actually is.

From the hex dump, the first G-frame is:

```
"$TG\xD0RX\x1E\e\xC4\x15\xFF\x003\x00\x00\x00#\xE1"
24 54 47 d0 52 58 1e 1b c4 15 ff 00 33 00 00 00 23 e1
```

Looking at byte offset  3-6 d0 52 58 1e , aka latitude:

```
$ irb
irb(main):001:0> puts 0x1e5852d0.to_f / 10000000
50.9104848
```

which is precisely what I'd expect.

You may also find the parseraw.rb script in the archive to replay elements (i.e. ltm) from the mwp raw log in real time.

**Comment by raul-ortega** [comment]
Thanks stronnag, there were an error in my code. Now lat/lon seem to be good. Altitude is always 0, and sats 17:

Lat: 50.910485, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910485, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910485, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910485, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910485, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
...
...
Lat: 50.910485, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910484, Lon: -1.535076, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910484, Lon: -1.535074, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910483, Lon: -1.535073, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910482, Lon: -1.535071, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910481, Lon: -1.535069, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910478, Lon: -1.535067, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910476, Lon: -1.535064, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910474, Lon: -1.535061, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910472, Lon: -1.535058, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910470, Lon: -1.535055, Alt: 0, Sats: 17, SatsFix: 71
Lat: 50.910468, Lon: -1.535052, Alt: 0, Sats: 17, SatsFix: 71
....

**Comment by stronnag** [comment]
The majority of the reports have non-zero altitude (between 1 and 10 metres). I don't think you're decoding the fix/sats correctly either.

There is a log of the complete decoded GFRAMES
http://www.zen35309.zen.co.uk/mwp-ltm/ltm_gframes.txt
and  the script I used to create this data
http://www.zen35309.zen.co.uk/mwp-ltm/test-ltm.rb from the raw byte stream

Examples:

```
lat: 50.910486 lon: -1.535077 speed 0 alt: 1.0 flags 23 (fix: 3 sats: 8)
...
lat: 50.910303 lon: -1.534951 speed 1 alt: 10.0 flags 27 (fix: 3 sats: 9)
```

Seems about right from what I recall, good fix and sats for PH / RTH testing

I will update the script for the other frame types, and reply here when it's done.

**Comment by stronnag** [comment]
OK, updated the ruby script decoder (same link as above), and decoded all frames  http://www.zen35309.zen.co.uk/mwp-ltm/ltm_allframes.txt. The letter at the begining is the frame type, so you can grep the type easily.

```
G= lat: 50.910260 lon: -1.534976 speed 2 alt: 9.0 flags 23 (fix: 3 sats: 8)
A= pitch: 3 roll: 0 head: 24
S= vbat 11.0 curr 0.0 rssi: 0 airspd 0 status: 0x0d
O= homelat: 50.910485 homelon: -1.535076 hfix: 1
A= pitch: 3 roll: 0 head: 25
G= lat: 50.910271 lon: -1.534968 speed 3 alt: 9.0 flags 23 (fix: 3 sats: 8)
A= pitch: 2 roll: 0 head: 25
S= vbat 11.0 curr 0.0 rssi: 0 airspd 0 status: 0x0d
A= pitch: 2 roll: 0 head: 25
G= lat: 50.910282 lon: -1.534961 speed 3 alt: 9.0 flags 27 (fix: 3 sats: 9)
A= pitch: 0 roll: 0 head: 25
S= vbat 11.0 curr 0.0 rssi: 0 airspd 0 status: 0x25
A= pitch: -3 roll: 0 head: 25
G= lat: 50.910292 lon: -1.534954 speed 3 alt: 9.0 flags 27 (fix: 3 sats: 9)
A= pitch: -9 roll: -1 head: 24
S= vbat 11.0 curr 0.0 rssi: 0 airspd 0 status: 0x25
A= pitch: -14 roll: 0 head: 24
G= lat: 50.910302 lon: -1.534947 speed 3 alt: 10.0 flags 27 (fix: 3 sats: 9)
A= pitch: -18 roll: -1 head: 24
S= vbat 11.0 curr 0.0 rssi: 0 airspd 0 status: 0x25
A= pitch: -20 roll: -1 head: 24
G= lat: 50.910311 lon: -1.534941 speed 4 alt: 10.0 flags 27 (fix: 3 sats: 9)
A= pitch: -21 roll: 0 head: 25
S= vbat 11.0 curr 0.0 rssi: 0 airspd 0 status: 0x25
O= homelat: 50.910485 homelon: -1.535076 hfix: 1
```

**Comment by raul-ortega** [comment]
I think is better now: 

Lat/Lon: 50.910485, -1.535076, Alt: 0.510000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.460000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.500000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.590000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.560000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.510000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.510000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.520000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.500000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.470000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.420000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.440000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.520000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.530000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.540000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.520000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.550000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535077, Alt: 0.600000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 0.540000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535077, Alt: 0.470000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535077, Alt: 0.480000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 0.500000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 0.480000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 0.470000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 0.430000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 0.460000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 0.650000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 1.110000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 1.490000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 1.630000, Sats: 8, FixType:3
Lat/Lon: 50.910486, -1.535077, Alt: 1.640000, Sats: 8, FixType:3
Lat/Lon: 50.910485, -1.535076, Alt: 1.690000, Sats: 8, FixType:3
Lat/Lon: 50.910484, -1.535076, Alt: 1.820000, Sats: 8, FixType:3
Lat/Lon: 50.910484, -1.535074, Alt: 2.040000, Sats: 8, FixType:3
Lat/Lon: 50.910483, -1.535073, Alt: 2.240000, Sats: 8, FixType:3
Lat/Lon: 50.910482, -1.535071, Alt: 2.520000, Sats: 8, FixType:3
Lat/Lon: 50.910481, -1.535069, Alt: 2.870000, Sats: 8, FixType:3
Lat/Lon: 50.910478, -1.535067, Alt: 3.270000, Sats: 8, FixType:3
Lat/Lon: 50.910476, -1.535064, Alt: 3.640000, Sats: 8, FixType:3
Lat/Lon: 50.910474, -1.535061, Alt: 4.050000, Sats: 9, FixType:3
Lat/Lon: 50.910472, -1.535058, Alt: 4.360000, Sats: 9, FixType:3
Lat/Lon: 50.910470, -1.535055, Alt: 4.660000, Sats: 9, FixType:3
Lat/Lon: 50.910468, -1.535052, Alt: 4.930000, Sats: 9, FixType:3
Lat/Lon: 50.910465, -1.535050, Alt: 5.190000, Sats: 9, FixType:3
Lat/Lon: 50.910461, -1.535047, Alt: 5.370000, Sats: 9, FixType:3
Lat/Lon: 50.910457, -1.535045, Alt: 5.560000, Sats: 9, FixType:3
Lat/Lon: 50.910451, -1.535043, Alt: 5.730000, Sats: 8, FixType:3
Lat/Lon: 50.910442, -1.535043, Alt: 5.860000, Sats: 8, FixType:3
Lat/Lon: 50.910434, -1.535044, Alt: 6.010000, Sats: 8, FixType:3
Lat/Lon: 50.910424, -1.535047, Alt: 6.180000, Sats: 8, FixType:3
Lat/Lon: 50.910413, -1.535051, Alt: 6.450000, Sats: 8, FixType:3
Lat/Lon: 50.910406, -1.535053, Alt: 6.760000, Sats: 8, FixType:3
Lat/Lon: 50.910400, -1.535055, Alt: 7.060000, Sats: 9, FixType:3
Lat/Lon: 50.910392, -1.535060, Alt: 7.270000, Sats: 9, FixType:3
Lat/Lon: 50.910383, -1.535067, Alt: 7.410000, Sats: 9, FixType:3
Lat/Lon: 50.910372, -1.535076, Alt: 7.460000, Sats: 9, FixType:3
Lat/Lon: 50.910362, -1.535084, Alt: 7.480000, Sats: 9, FixType:3
Lat/Lon: 50.910351, -1.535095, Alt: 7.510000, Sats: 9, FixType:3
Lat/Lon: 50.910341, -1.535104, Alt: 7.610000, Sats: 9, FixType:3
Lat/Lon: 50.910331, -1.535112, Alt: 7.740000, Sats: 9, FixType:3
Lat/Lon: 50.910321, -1.535121, Alt: 7.880000, Sats: 9, FixType:3
Lat/Lon: 50.910310, -1.535128, Alt: 8.010000, Sats: 9, FixType:3
Lat/Lon: 50.910298, -1.535136, Alt: 8.170000, Sats: 9, FixType:3
Lat/Lon: 50.910287, -1.535140, Alt: 8.350000, Sats: 9, FixType:3
Lat/Lon: 50.910276, -1.535146, Alt: 8.490000, Sats: 9, FixType:3
Lat/Lon: 50.910265, -1.535150, Alt: 8.540000, Sats: 9, FixType:3
Lat/Lon: 50.910252, -1.535147, Alt: 8.550000, Sats: 9, FixType:3
Lat/Lon: 50.910238, -1.535143, Alt: 8.580000, Sats: 8, FixType:3
Lat/Lon: 50.910226, -1.535135, Alt: 8.640000, Sats: 8, FixType:3
Lat/Lon: 50.910215, -1.535123, Alt: 8.710000, Sats: 8, FixType:3
Lat/Lon: 50.910205, -1.535111, Alt: 8.840000, Sats: 8, FixType:3
Lat/Lon: 50.910199, -1.535100, Alt: 9.060000, Sats: 8, FixType:3
Lat/Lon: 50.910190, -1.535089, Alt: 9.220000, Sats: 7, FixType:3
Lat/Lon: 50.910182, -1.535079, Alt: 9.230000, Sats: 7, FixType:3
Lat/Lon: 50.910179, -1.535070, Alt: 9.190000, Sats: 8, FixType:3
Lat/Lon: 50.910175, -1.535064, Alt: 9.320000, Sats: 8, FixType:3
Lat/Lon: 50.910175, -1.535060, Alt: 9.540000, Sats: 8, FixType:3
Lat/Lon: 50.910171, -1.535053, Alt: 9.850000, Sats: 8, FixType:3
Lat/Lon: 50.910169, -1.535046, Alt: 10.050000, Sats: 8, FixType:3
Lat/Lon: 50.910174, -1.535035, Alt: 10.080000, Sats: 8, FixType:3
Lat/Lon: 50.910175, -1.535030, Alt: 9.960000, Sats: 7, FixType:3
Lat/Lon: 50.910185, -1.535025, Alt: 9.830000, Sats: 7, FixType:3
Lat/Lon: 50.910196, -1.535018, Alt: 9.700000, Sats: 8, FixType:3
Lat/Lon: 50.910213, -1.535008, Alt: 9.590000, Sats: 8, FixType:3
Lat/Lon: 50.910227, -1.534998, Alt: 9.510000, Sats: 8, FixType:3
Lat/Lon: 50.910238, -1.534991, Alt: 9.500000, Sats: 8, FixType:3
Lat/Lon: 50.910249, -1.534984, Alt: 9.470000, Sats: 8, FixType:3
Lat/Lon: 50.910260, -1.534976, Alt: 9.530000, Sats: 8, FixType:3
Lat/Lon: 50.910271, -1.534968, Alt: 9.650000, Sats: 8, FixType:3
Lat/Lon: 50.910282, -1.534961, Alt: 9.730000, Sats: 9, FixType:3
Lat/Lon: 50.910292, -1.534954, Alt: 9.950000, Sats: 9, FixType:3
Lat/Lon: 50.910302, -1.534947, Alt: 10.170000, Sats: 9, FixType:3
Lat/Lon: 50.910311, -1.534941, Alt: 10.380000, Sats: 9, FixType:3
Lat/Lon: 50.910319, -1.534935, Alt: 10.510000, Sats: 9, FixType:3
Lat/Lon: 50.910325, -1.534930, Alt: 10.590000, Sats: 9, FixType:3
Lat/Lon: 50.910329, -1.534928, Alt: 10.600000, Sats: 9, FixType:3
Lat/Lon: 50.910331, -1.534927, Alt: 10.590000, Sats: 9, FixType:3
Lat/Lon: 50.910333, -1.534927, Alt: 10.580000, Sats: 9, FixType:3
Lat/Lon: 50.910333, -1.534928, Alt: 10.570000, Sats: 9, FixType:3
Lat/Lon: 50.910333, -1.534929, Alt: 10.460000, Sats: 9, FixType:3
Lat/Lon: 50.910332, -1.534929, Alt: 10.360000, Sats: 9, FixType:3
Lat/Lon: 50.910330, -1.534932, Alt: 10.390000, Sats: 9, FixType:3
Lat/Lon: 50.910328, -1.534935, Alt: 10.430000, Sats: 9, FixType:3
Lat/Lon: 50.910325, -1.534937, Alt: 10.460000, Sats: 9, FixType:3
Lat/Lon: 50.910322, -1.534940, Alt: 10.460000, Sats: 9, FixType:3
Lat/Lon: 50.910319, -1.534942, Alt: 10.470000, Sats: 9, FixType:3
Lat/Lon: 50.910315, -1.534944, Alt: 10.490000, Sats: 9, FixType:3
Lat/Lon: 50.910312, -1.534946, Alt: 10.490000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534948, Alt: 10.460000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534949, Alt: 10.410000, Sats: 9, FixType:3
Lat/Lon: 50.910303, -1.534951, Alt: 10.350000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534953, Alt: 10.250000, Sats: 9, FixType:3
Lat/Lon: 50.910296, -1.534955, Alt: 10.140000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534956, Alt: 9.950000, Sats: 9, FixType:3
Lat/Lon: 50.910289, -1.534958, Alt: 9.710000, Sats: 9, FixType:3
Lat/Lon: 50.910286, -1.534959, Alt: 9.390000, Sats: 9, FixType:3
Lat/Lon: 50.910284, -1.534961, Alt: 9.230000, Sats: 9, FixType:3
Lat/Lon: 50.910281, -1.534962, Alt: 9.090000, Sats: 9, FixType:3
Lat/Lon: 50.910279, -1.534963, Alt: 8.940000, Sats: 9, FixType:3
Lat/Lon: 50.910277, -1.534963, Alt: 8.830000, Sats: 9, FixType:3
Lat/Lon: 50.910275, -1.534963, Alt: 8.840000, Sats: 9, FixType:3
Lat/Lon: 50.910274, -1.534963, Alt: 8.950000, Sats: 9, FixType:3
Lat/Lon: 50.910273, -1.534962, Alt: 9.090000, Sats: 9, FixType:3
Lat/Lon: 50.910272, -1.534962, Alt: 9.270000, Sats: 9, FixType:3
Lat/Lon: 50.910271, -1.534961, Alt: 9.390000, Sats: 9, FixType:3
Lat/Lon: 50.910270, -1.534960, Alt: 9.410000, Sats: 9, FixType:3
Lat/Lon: 50.910270, -1.534958, Alt: 9.470000, Sats: 9, FixType:3
Lat/Lon: 50.910269, -1.534957, Alt: 9.500000, Sats: 9, FixType:3
Lat/Lon: 50.910269, -1.534955, Alt: 9.460000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534954, Alt: 9.430000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534952, Alt: 9.480000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534951, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534950, Alt: 9.510000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534949, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534948, Alt: 9.580000, Sats: 9, FixType:3
Lat/Lon: 50.910268, -1.534948, Alt: 9.640000, Sats: 9, FixType:3
Lat/Lon: 50.910269, -1.534947, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910269, -1.534947, Alt: 9.790000, Sats: 9, FixType:3
Lat/Lon: 50.910270, -1.534947, Alt: 9.840000, Sats: 9, FixType:3
Lat/Lon: 50.910270, -1.534947, Alt: 9.850000, Sats: 9, FixType:3
Lat/Lon: 50.910271, -1.534948, Alt: 9.810000, Sats: 9, FixType:3
Lat/Lon: 50.910272, -1.534948, Alt: 9.820000, Sats: 9, FixType:3
Lat/Lon: 50.910273, -1.534949, Alt: 9.860000, Sats: 9, FixType:3
Lat/Lon: 50.910274, -1.534950, Alt: 9.880000, Sats: 9, FixType:3
Lat/Lon: 50.910275, -1.534951, Alt: 9.890000, Sats: 9, FixType:3
Lat/Lon: 50.910276, -1.534952, Alt: 9.830000, Sats: 9, FixType:3
Lat/Lon: 50.910276, -1.534952, Alt: 9.780000, Sats: 9, FixType:3
Lat/Lon: 50.910277, -1.534953, Alt: 9.730000, Sats: 9, FixType:3
Lat/Lon: 50.910278, -1.534954, Alt: 9.670000, Sats: 9, FixType:3
Lat/Lon: 50.910278, -1.534955, Alt: 9.620000, Sats: 9, FixType:3
Lat/Lon: 50.910279, -1.534956, Alt: 9.490000, Sats: 9, FixType:3
Lat/Lon: 50.910280, -1.534957, Alt: 9.390000, Sats: 9, FixType:3
Lat/Lon: 50.910280, -1.534957, Alt: 9.370000, Sats: 9, FixType:3
Lat/Lon: 50.910280, -1.534957, Alt: 9.300000, Sats: 9, FixType:3
Lat/Lon: 50.910281, -1.534957, Alt: 9.280000, Sats: 9, FixType:3
Lat/Lon: 50.910281, -1.534958, Alt: 9.290000, Sats: 9, FixType:3
Lat/Lon: 50.910281, -1.534957, Alt: 9.310000, Sats: 9, FixType:3
Lat/Lon: 50.910282, -1.534957, Alt: 9.390000, Sats: 9, FixType:3
Lat/Lon: 50.910282, -1.534957, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910282, -1.534957, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910282, -1.534957, Alt: 9.740000, Sats: 9, FixType:3
Lat/Lon: 50.910282, -1.534958, Alt: 9.790000, Sats: 9, FixType:3
Lat/Lon: 50.910283, -1.534958, Alt: 9.820000, Sats: 9, FixType:3
Lat/Lon: 50.910283, -1.534958, Alt: 9.790000, Sats: 9, FixType:3
Lat/Lon: 50.910283, -1.534959, Alt: 9.740000, Sats: 9, FixType:3
Lat/Lon: 50.910283, -1.534959, Alt: 9.780000, Sats: 9, FixType:3
Lat/Lon: 50.910283, -1.534959, Alt: 9.830000, Sats: 9, FixType:3
Lat/Lon: 50.910284, -1.534959, Alt: 9.840000, Sats: 9, FixType:3
Lat/Lon: 50.910284, -1.534960, Alt: 9.790000, Sats: 9, FixType:3
Lat/Lon: 50.910285, -1.534960, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910285, -1.534960, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910286, -1.534960, Alt: 9.680000, Sats: 9, FixType:3
Lat/Lon: 50.910286, -1.534960, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910287, -1.534960, Alt: 9.610000, Sats: 9, FixType:3
Lat/Lon: 50.910287, -1.534960, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910288, -1.534960, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910289, -1.534960, Alt: 9.640000, Sats: 9, FixType:3
Lat/Lon: 50.910289, -1.534959, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910290, -1.534959, Alt: 9.680000, Sats: 9, FixType:3
Lat/Lon: 50.910290, -1.534959, Alt: 9.670000, Sats: 9, FixType:3
Lat/Lon: 50.910291, -1.534958, Alt: 9.650000, Sats: 9, FixType:3
Lat/Lon: 50.910291, -1.534958, Alt: 9.670000, Sats: 9, FixType:3
Lat/Lon: 50.910291, -1.534957, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910292, -1.534957, Alt: 9.680000, Sats: 9, FixType:3
Lat/Lon: 50.910292, -1.534956, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910292, -1.534955, Alt: 9.710000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534954, Alt: 9.770000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534953, Alt: 9.820000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534952, Alt: 9.860000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534951, Alt: 9.850000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534950, Alt: 9.800000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534949, Alt: 9.740000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534949, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534948, Alt: 9.740000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.760000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534945, Alt: 9.680000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534945, Alt: 9.650000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534945, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534945, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.810000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.840000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.860000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.800000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.730000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910293, -1.534944, Alt: 9.700000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534945, Alt: 9.710000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534945, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534945, Alt: 9.560000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534945, Alt: 9.540000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.490000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.510000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.500000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.530000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.580000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.540000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.590000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.580000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534947, Alt: 9.600000, Sats: 9, FixType:3
Lat/Lon: 50.910294, -1.534946, Alt: 9.610000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534946, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534946, Alt: 9.650000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534946, Alt: 9.570000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534946, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534945, Alt: 9.540000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534945, Alt: 9.640000, Sats: 9, FixType:3
Lat/Lon: 50.910295, -1.534945, Alt: 9.660000, Sats: 9, FixType:3
Lat/Lon: 50.910296, -1.534945, Alt: 9.700000, Sats: 9, FixType:3
Lat/Lon: 50.910296, -1.534944, Alt: 9.760000, Sats: 9, FixType:3
Lat/Lon: 50.910296, -1.534944, Alt: 9.790000, Sats: 9, FixType:3
Lat/Lon: 50.910297, -1.534944, Alt: 9.870000, Sats: 9, FixType:3
Lat/Lon: 50.910297, -1.534944, Alt: 9.950000, Sats: 9, FixType:3
Lat/Lon: 50.910298, -1.534944, Alt: 9.940000, Sats: 9, FixType:3
Lat/Lon: 50.910298, -1.534943, Alt: 9.900000, Sats: 9, FixType:3
Lat/Lon: 50.910299, -1.534943, Alt: 9.850000, Sats: 9, FixType:3
Lat/Lon: 50.910299, -1.534943, Alt: 9.770000, Sats: 9, FixType:3
Lat/Lon: 50.910299, -1.534942, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910299, -1.534942, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534941, Alt: 9.620000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534940, Alt: 9.600000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534939, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534939, Alt: 9.590000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534938, Alt: 9.560000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534937, Alt: 9.590000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534937, Alt: 9.660000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.670000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.700000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.790000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534936, Alt: 9.660000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534937, Alt: 9.620000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534937, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534938, Alt: 9.600000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534939, Alt: 9.550000, Sats: 9, FixType:3
Lat/Lon: 50.910300, -1.534939, Alt: 9.540000, Sats: 9, FixType:3
Lat/Lon: 50.910301, -1.534940, Alt: 9.490000, Sats: 9, FixType:3
Lat/Lon: 50.910301, -1.534941, Alt: 9.470000, Sats: 9, FixType:3
Lat/Lon: 50.910301, -1.534942, Alt: 9.420000, Sats: 9, FixType:3
Lat/Lon: 50.910301, -1.534943, Alt: 9.390000, Sats: 9, FixType:3
Lat/Lon: 50.910302, -1.534944, Alt: 9.380000, Sats: 9, FixType:3
Lat/Lon: 50.910302, -1.534944, Alt: 9.380000, Sats: 9, FixType:3
Lat/Lon: 50.910302, -1.534945, Alt: 9.420000, Sats: 9, FixType:3
Lat/Lon: 50.910302, -1.534946, Alt: 9.470000, Sats: 9, FixType:3
Lat/Lon: 50.910303, -1.534947, Alt: 9.480000, Sats: 9, FixType:3
Lat/Lon: 50.910303, -1.534948, Alt: 9.560000, Sats: 9, FixType:3
Lat/Lon: 50.910303, -1.534949, Alt: 9.650000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534949, Alt: 9.660000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534950, Alt: 9.670000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534951, Alt: 9.710000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534951, Alt: 9.690000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534952, Alt: 9.670000, Sats: 8, FixType:3
Lat/Lon: 50.910306, -1.534952, Alt: 9.730000, Sats: 8, FixType:3
Lat/Lon: 50.910306, -1.534953, Alt: 9.740000, Sats: 8, FixType:3
Lat/Lon: 50.910307, -1.534953, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534954, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534954, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.710000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.780000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.810000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.770000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534956, Alt: 9.730000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.720000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.770000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534955, Alt: 9.850000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534955, Alt: 9.940000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534955, Alt: 10.010000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534955, Alt: 9.950000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 9.910000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 9.840000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 9.680000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534955, Alt: 9.530000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534955, Alt: 9.350000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534955, Alt: 9.130000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 8.950000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 8.890000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 8.860000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534955, Alt: 8.850000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534956, Alt: 8.880000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534956, Alt: 9.020000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534956, Alt: 9.180000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534956, Alt: 9.330000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534955, Alt: 9.540000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.650000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534955, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534954, Alt: 9.890000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534953, Alt: 9.950000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534952, Alt: 9.980000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534951, Alt: 10.000000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534950, Alt: 10.040000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534949, Alt: 10.010000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534948, Alt: 9.920000, Sats: 9, FixType:3
Lat/Lon: 50.910309, -1.534947, Alt: 9.860000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534946, Alt: 9.830000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534945, Alt: 9.810000, Sats: 9, FixType:3
Lat/Lon: 50.910308, -1.534945, Alt: 9.820000, Sats: 9, FixType:3
Lat/Lon: 50.910307, -1.534945, Alt: 9.830000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534944, Alt: 9.750000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534945, Alt: 9.670000, Sats: 9, FixType:3
Lat/Lon: 50.910306, -1.534945, Alt: 9.630000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534945, Alt: 9.600000, Sats: 9, FixType:3
Lat/Lon: 50.910305, -1.534946, Alt: 9.530000, Sats: 9, FixType:3
Lat/Lon: 50.910304, -1.534946, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910303, -1.534947, Alt: 9.520000, Sats: 9, FixType:3
Lat/Lon: 50.910302, -1.534947, Alt: 9.480000, Sats: 9, FixType:3
...

**Comment by stronnag** [comment]
looking good

**Comment by raul-ortega** [comment]
I'm too tired today, I waked up at 6, and all the day working...I'll integrate it in the antenna tracker code tomorrow. I think it works now.

**Comment by raul-ortega** [comment]
v0.8 uploaded, now it supports LTM protocol. Tested at 9600 baud and working well.
#define LTM added to config.h
Enjoy it...

**Comment by marksev1** [comment]
Great, thanks! Btw in config.h i can change baud rate to 2400 right?
